src/client/components/Companies/forms/companies.js

```diff
@@ -0,0 +1,68 @@
+const fields = {
+  types: {
+    key: 'Types',
+    label: 'Types',
+    type: 'select',
+    domainValues: ['Client', 'Partner', 'Tenant'],
```

Comment:
doesn't work must have [value, label] where label is rendered, value will be stored